core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -4048,6 +4048,18 @@ private Tester getExtendedTester() {
     sql(sql).trim(true).ok();
   }
 
+  /**
+   * Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-4295">[CALCITE-4295]
+   * Composite of two checker with SqlOperandCountRange throws IllegalArgumentException</a>.
```

Comment:
The title in the comment reflects the "old" title of the ticket, could it be updated?